doc/command_ref.rst

```diff
@@ -1061,7 +1061,7 @@ Remove Command
     dnf-shell sub-commands could help.
 
 ``dnf [options] remove --oldinstallonly``
-    Removes old installonly packages, keeping only ``installonly_limit`` latest versions.
+    Removes old installonly packages, keeping only latest versions and version of running kernel.
```

Comment:
With your patch, only one latest version will be kept, right? Can you change the plural to singular, so that it's clear from the documentation?